Sources/PackageModel/Module.swift

```diff
@@ -16,6 +16,7 @@
 */
 
 @_exported import enum PackageDescription.SystemPackageProvider
+import struct Basic.RelativePath
```

Comment:
Cool, first official use!  I'll be posting PRs to convert over the whole code base, a piece at a time.